src/screens/NewsScreen/index.js

```diff
@@ -5,6 +5,7 @@ import {
   FlatList,
   InteractionManager,
   Platform,
+  RefreshControl,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `RefreshControl` component has been imported but it's not clear if it was previously available in the project. If it wasn't, ensure that it is installed and properly configured to avoid any runtime errors.
```diff
+   RefreshControl,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->